pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -223,7 +230,7 @@ export class Terminal extends React.Component {
 
     componentWillUnmount() {
         this.disconnectChannel();
-        this.state.terminal.dispose();
+        this.terminal.dispose();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18157-20230112-171828-95ce097c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)